engine-precompiles/src/native.rs

```diff
@@ -323,8 +337,8 @@ impl<I: IO> Precompile for ExitToNear<I> {
                 let erc20_address = context.caller;
                 let nep141_address = get_nep141_from_erc20(erc20_address.as_bytes(), &self.io)?;
 
-                let amount = U256::from_big_endian(&input[..32]);
-                input = &input[32..];
+                let amount = U256::from_big_endian(input.get(..32).unwrap());
+                input = input.get(32..).unwrap();
```

Comment:
Do not use `unwrap()`. One of the Aurora Audit proposals was to avoid `unwrap`.

Reply:
Just a bit of guidance, if you are 100% sure that it can not possibly fail, `.expect("Some explanation")` is acceptable, otherwise, error return.